packages/cfpb-layout/src/molecules/card.less

```diff
@@ -0,0 +1,114 @@
+.m-card {
+    width: 100%;
+    background: @white;
+
+    // Both regular and featured cards.
+    &:not( .m-card__featured ),
+    & > a {
+        display: flex;
+        flex-direction: column;
+        flex-grow: 1;
+        flex-basis: 0;
+        box-sizing: border-box;
+        border: 1px solid @gray-20;
+        border-bottom-width: 3px;
+        padding: unit( @grid_gutter-width / @base-font-size-px, em );
+        height: 100%;
+    }
+
+    & a:focus {
+        outline-offset: 2px;
+    }
+
+    &_footer {
+        margin-top: auto;
+    }
+
+    &_footer > a {
+        font-weight: 500;
+        border-bottom-width: 1px;
+    }
+
+    // Shrink heading at smaller screen sizes.
+    &_heading {
+        .respond-to-max( @bp-med-min, {
+            .h3();
+        } );
+    }
+
+    // Regular cards.
+    &_heading > a {
+        color: @black;
+
+        &:hover {
+            color: @pacific;
+        }
+
+        & .m-card_icon {
+            font-size: 1em;
+            margin-bottom: 0;
+            padding-right: unit( 7.5px / @base-font-size-px, em );
+        }
+
+        & .m-card_icon,
+        & span {
+            display: table-cell;
+        }
+    }
+
+    & > .m-list {
+        margin-top: 10px;
+        margin-bottom: 30px;
+    }
+
+    // Featured cards.
+    &__featured {
+
+        .m-card_heading {
+            margin: 0;
+        }
+
+        .m-card_icon {
+            font-size: unit( 30px / @base-font-size-px, em );
+            margin-bottom: unit( 5px / @base-font-size-px, em );
+        }
+
+        .m-card_icon,
+        p {
+            color: @black;
+        }
+
+        &:hover {
+            box-shadow: 0 8px 0 0 inset @green,
+                        2px 0 0 0 inset @gray-20,
+                        -2px 0 0 0 inset @gray-20;
+        }
+
+        &:hover .m-card_footer > span {
+            border-style: solid;
+            color: @dark-pacific;
+        }
+
+        .m-card_footer > span {
+            display: inline-block;
```

Comment:
Any chance we can keep this span inline to prevent the border from pulling away from the text if the line breaks?

<img width="726" alt="Screen Shot 2020-02-28 at 4 40 57 AM" src="https://user-images.githubusercontent.com/1060248/75537381-e6ee1680-59e4-11ea-9b58-c4495fcd600c.png">

Reply:
Fixed in https://github.com/cfpb/design-system/pull/428/commits/37567d1eab58379c383f074c27073bb5793410e7

Reply:
Changed in https://github.com/cfpb/design-system/pull/428/commits/37567d1eab58379c383f074c27073bb5793410e7